### PR TITLE
Translated Conditional and List.

### PR DESCRIPTION
**Pull request by lagp0310**
## Description of Problem

None.

## Proposed Solution

Translated Files:

- `src/guide/conditional.md`.
- `src/guide/list.md`.

## Additional Information

None.

### CODE REVIEW

File: src/guide/conditional.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
-# Conditional Rendering
+# Renderización Condicional
 
 ## `v-if`
 
-The directive `v-if` is used to conditionally render a block. The block will only be rendered if the directive's expression returns a truthy value.
+La directiva `v-if` es utilizada para renderizar condicionalmente un bloque de código. El bloque de código solo será renderizado si la expresión de la directiva retorna un valor de verdad.
```
Comment (migsalazar) [inline src/guide/conditional.md]: What do you think about this?

```suggestion
La directiva `v-if` es utilizada para renderizar condicionalmente un bloque de código. El bloque de código solo será renderizado si la expresión de la directiva retorna un valor verdadero.
```

File: src/guide/conditional.md [hunk 2]
```diff
@@ -57,38 +57,38 @@ The `v-else-if`, as the name suggests, serves as an "else if block" for `v-if`.
...
-`v-show` doesn't support the `<template>` element, nor does it work with `v-else`.
+`v-show` no es compatible con elementos `<template>`, ni tampoco funciona con `v-else`.
 
 ## `v-if` vs `v-show`
 
-`v-if` is "real" conditional rendering because it ensures that event listeners and child components inside the conditional block are properly destroyed and re-created during toggles.
+`v-if` es una renderización condicional “real” ya que garantiza que los eventos y componentes secundarios dentro del bloque condicional sean debidamente destruidos y recreados durante la alternancia.
 
-`v-if` is also **lazy**: if the condition is false on initial render, it will not do anything - the conditional block won't be rendered until the condition becomes true for the first time.
+`v-if` también es **_lazy_**: si la condición es falsa en la representación inicial, no se hará nada. El bloque condicional no se procesará hasta que la condición se convierta en true por primera vez.
```
Comment (migsalazar) [inline src/guide/conditional.md]: How should we use the translation of "render"? I have used it as "renderizar". @lagp0310 @Tintef @sxcamacho 

```suggestion
`v-if` también es **_lazy_**: si la condición es falsa en la renderización inicial, no se hará nada. El bloque condicional no se procesará hasta que la condición se convierta en `true` por primera vez.
```

File: src/guide/list.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-# List Rendering
+# Renderizado de Lista
 
-## Mapping an Array to Elements with `v-for`
+## Mapeando una matriz a Elementos con `v-for`
```
Comment (migsalazar) [inline src/guide/list.md]: What do you think about this?

```suggestion
## Mapeando un Arreglo a Elementos con `v-for`
```

File: src/guide/list.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 
-We can use the `v-for` directive to render a list of items based on an array. The `v-for` directive requires a special syntax in the form of `item in items`, where `items` is the source data array and `item` is an **alias** for the array element being iterated on:
+Podemos usar la directiva `v-for` para renderizar una lista de elementos basada en una matriz. La directiva `v-for` requiere una sintaxis especial en forma de `item in items`, donde los `items` son la matriz de datos de origen y el `item` es un **alias** para el elemento de matriz que se está iterando:
```
Comment (migsalazar) [inline src/guide/list.md]: Same here -as my previous comment-

```suggestion
Podemos usar la directiva `v-for` para renderizar una lista de elementos basada en un arreglo. La directiva `v-for` requiere una sintaxis especial en forma de `item in items`, donde los `items` son el arreglo de datos de origen y el `item` es un **alias** para el elemento de matriz que se está iterando:
```

File: src/guide/conditional.md (hunk 2 above)
Comment (Tintef) [reply src/guide/conditional.md]: I think `renderiazado/renderizar/renderización` are good

File: src/guide/conditional.md [hunk 5]
```diff
@@ -57,38 +57,38 @@ The `v-else-if`, as the name suggests, serves as an "else if block" for `v-if`.
...
 </div>
 ```
 
-Similar to `v-else`, a `v-else-if` element must immediately follow a `v-if` or a `v-else-if` element.
+Similar a `v-else`, un elemento `v-else-if` debe seguir inmediatamente a un elemento `v-if` o `v-else-if`.
 
 ## `v-show`
 
-Another option for conditionally displaying an element is the `v-show` directive. The usage is largely the same:
+Otra opción para mostrar un elemento condicionalmente es la directiva `v-show`. La utilización es básicamente la misma:
```
Comment (Tintef) [inline src/guide/conditional.md]: What do you think about this?
```suggestion
Otra opción para mostrar un elemento condicionalmente es la directiva `v-show`. El uso es básicamente el mismo:
```

File: src/guide/conditional.md [hunk 6]
```diff
@@ -57,38 +57,38 @@ The `v-else-if`, as the name suggests, serves as an "else if block" for `v-if`.
...
-The difference is that an element with `v-show` will always be rendered and remain in the DOM; `v-show` only toggles the `display` CSS property of the element.
+La diferencia es que un elemento con `v-show` siempre se renderizará y permanecerá en el DOM; `v-show` simplemente alterna la propiedad CSS `display` del elemento.
 
-`v-show` doesn't support the `<template>` element, nor does it work with `v-else`.
+`v-show` no es compatible con elementos `<template>`, ni tampoco funciona con `v-else`.
 
 ## `v-if` vs `v-show`
 
-`v-if` is "real" conditional rendering because it ensures that event listeners and child components inside the conditional block are properly destroyed and re-created during toggles.
+`v-if` es una renderización condicional “real” ya que garantiza que los eventos y componentes secundarios dentro del bloque condicional sean debidamente destruidos y recreados durante la alternancia.
```
Comment (Tintef) [inline src/guide/conditional.md]: I think `recreados` is not a good option here. What do you think about this?
```suggestion
`v-if` es una renderización condicional “real” ya que garantiza que los eventos y componentes secundarios dentro del bloque condicional sean debidamente destruidos y creados nuevamente durante la alternancia.
```

File: src/guide/conditional.md [hunk 7]
```diff
@@ -57,38 +57,38 @@ The `v-else-if`, as the name suggests, serves as an "else if block" for `v-if`.
...
 
-In comparison, `v-show` is much simpler - the element is always rendered regardless of initial condition, with CSS-based toggling.
+En comparación, `v-show` es mucho más simple: el elemento siempre se representa independientemente de la condición inicial, con alternancia basada en CSS.
 
-Generally speaking, `v-if` has higher toggle costs while `v-show` has higher initial render costs. So prefer `v-show` if you need to toggle something very often, and prefer `v-if` if the condition is unlikely to change at runtime.
+En general, `v-if` tiene costos de alternancia más altos, mientras que `v-show` tiene costos de renderización iniciales más altos. Por lo tanto, prefiera `v-show` si necesita cambiar algo muy a menudo, y prefiera `v-if` si es poco probable que la condición cambie en el tiempo de ejecución.
 
 ## `v-if` with `v-for`
 
 ::: tip Note
```
Comment (Tintef) [inline src/guide/conditional.md]: I think you forgot about this one 😅

File: src/guide/list.md [hunk 8]
```diff
@@ -132,39 +132,39 @@ And another for the index:
...
-Don't use non-primitive values like objects and arrays as `v-for` keys. Use string or numeric values instead.
+:::tip Nota
+No utilice valores no primitivos como objetos y vectores como clave para `v-for`. Utilice _string_ o valores numéricos en su lugar.
 :::
 
-For detailed usage of the `key` attribute, please see the [`key` API documentation](../api/special-attributes.html#key).
+Para ver el uso en detalle del atributo `key`, por favor vea la [documentación de la API para `key`](../api/special-attributes.html#key).
 
-## Array Change Detection
+## Detección del cambios en Array
```
Comment (Tintef) [inline src/guide/list.md]:
```suggestion
## Detección del cambios en Arreglos
```

File: src/guide/list.md [hunk 9]
```diff
@@ -174,23 +174,23 @@ Vue wraps an observed array's mutation methods so they will also trigger view up
 - `sort()`
 - `reverse()`
 
-You can open the console and play with the previous examples' `items` array by calling their mutation methods. For example: `example1.items.push({ message: 'Baz' })`.
+Puede abrir la consola y probar con la matriz de `items` de los ejemplos anteriores llamando a sus métodos de mutación. Por ejemplo: `example1.items.push({ message: 'Baz' })`.
 
-### Replacing an Array
+### Mutando un Array
```
Comment (Tintef) [inline src/guide/list.md]:
```suggestion
### Mutando un Arreglo
```

**Review by Tintef — COMMENTED** [review]
Great work 💪

File: src/guide/conditional.md (hunk 6 above)
Comment (lagp0310) [reply src/guide/conditional.md]: Yes, `creados nuevamente` it's a better choice for this. :smile:

File: src/guide/conditional.md (hunk 7 above)
Comment (lagp0310) [inline src/guide/conditional.md]:
```suggestion
::: tip Nota
```

Comment (lagp0310) [reply src/guide/conditional.md]: Haha, thanks! :smile:

File: src/guide/list.md (hunk 3 above)
Comment (lagp0310) [reply src/guide/list.md]: Thanks, @migsalazar! I forgot we're using `arreglo` instead of `matriz`.

**Comment by lagp0310** [comment]
> Great work

Thanks @Tintef! :+1: :smile:

**Pull request merged**